cli/src/maintenance.rs

```diff
@@ -692,6 +717,32 @@ impl SolidoState {
         None
     }
 
+    /// Try to sell the extra stSOL rewards for UST tokens.
+    pub fn try_sell_anker_rewards(&self) -> Option<(Instruction, MaintenanceOutput)> {
+        let anker_state = self.anker_state.as_ref()?;
+        let reserve_st_sol = anker_state.st_sol_reserve_balance;
+        let st_sol_amount = self
+            .solido
+            .exchange_rate
+            .exchange_sol(Lamports(anker_state.b_sol_total_supply_amount.0))
+            .unwrap();
```

Comment:
Note to self: this unwrap happens only if the rational multiplication in `exchange_sol` overflows. In the first epoch with zero deposits, it should still work fine.

@enriquefynn can you add a comment to clarify this, and use `.expect()` instead of `.unwrap()` to also codify that reason? (And also to avoid people saying “unwrap is baaad, mkay?”)